ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTPayload.scala

```diff
@@ -103,6 +90,8 @@ object AuthServiceJWTCodec {
   // ------------------------------------------------------------------------------------------------------------------
   // OpenID Connect (OIDC) namespace for custom JWT claims
   final val oidcNamespace: String = "https://daml.com/ledger-api"
+  // Unique scope for standard tokens, following the pattern of https://developers.google.com/identity/protocols/oauth2/scopes
+  final val scopeLedgerApiFull: String = "https://daml.com/ledger-api.full_control"
```

Comment:
Have you checked if you can set this from auth0, okta and maybe some other IAMs? iirc the problem with the old format on okta was that they didn’t allow urls in the place we had it. Haven’t checked if this is an issue here but better to check before than discover that it won’t work later.